boot-data-neo4j/src/test/java/com/example/demo/PostRepositoryWithTestContainersTest.java

```diff
@@ -29,26 +24,11 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @SpringBootTest
-//@DataNeo4jTest
-@Testcontainers
 @Slf4j
-// Testcontainers does not work well with per_class testinstance.
-// see: https://stackoverflow.com/questions/61357116/exception-mapped-port-can-only-be-obtained-after-the-container-is-started-when/61358336#61358336
-@TestInstance(TestInstance.Lifecycle.PER_METHOD)
 @ActiveProfiles("test")
+@Import(TestDemoApplication.class)
```

Comment:
We can specify the class attribute of `@SpringBootTest` instead of `@Import` here.

Reply:
Fixed